ReportPortal.ATM/LoggerLibrary/Interfaces/Loggers/ILoggerService.cs

```diff
@@ -8,6 +8,7 @@ public interface ILoggerService
         void LogCritical(Exception exception, string message, params object?[]? args);
         void LogDebug(string message, params object?[]? args);
         void LogError(Exception exception, string message, params object?[]? args);
+        void LogError(string message, params object?[]? args);
```

Comment:
You can make the `Exception exception` in the `LogError` method nullable, so that you do not need an overload.